docs/conf.py

```diff
@@ -38,19 +39,41 @@ def setup(app):
 version = package.__version__.split("-", 1)[0]
 release = package.__version__
 
+# Configuration for intersphinx: refer to the Python standard library.
+intersphinx_mapping = {
+    "python": ("https://docs.python.org/3/", None),
+    "numpy": ("https://numpy.org/devdocs", None),
+    "scipy": ("http://scipy.github.io/devdocs", None),
+    "matplotlib": ("https://matplotlib.org/stable", None),
+    "gwcs": ("https://gwcs.readthedocs.io/en/latest/", None),
+    "astropy": ("https://docs.astropy.org/en/stable/", None),
+    "stdatamodels": ("https://stdatamodels.readthedocs.io/en/latest/", None),
```

Comment:
There shouldn't be any dependence on stdatamodels. Is there?

Reply:
I'm not sure why that was there, but it was removed.